software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/riak/RiakNode.java

```diff
@@ -107,10 +107,10 @@
 
     // NB these two needed for clients to access
     @SetFromFlag("riakWebPort")
-    PortAttributeSensorAndConfigKey RIAK_WEB_PORT = new PortAttributeSensorAndConfigKey("riak.webPort", "Riak Web Port", "8098+");
+    PortAttributeSensorAndConfigKey RIAK_WEB_PORT = ConfigKeys.newPortSensorAndConfigKey("riak.web.port", "Riak Web Port", "8098+");
```

Comment:
I'd be very cautious: I'd create a second config key with the old name, and fallback to that. And I'd make sure a sensor with the old name is published.

Looking at some customer bespoke code, they depend on "riak.pbPort" and "riak.webPort" being published as sensors (for their enricher that propagates the public endpoint). We can tell them to change this, but we should not break backwards compatibility lightly.